app/Http/Controllers/Admin/StationCrudController.php

```diff
@@ -117,6 +118,22 @@ public function setup()
                 'type' => 'number',
                 'attributes' => ["step" => "any"], // allow decimals
             ],
+            [
+                'name' => 'filename_map',
+                'label' => 'Map Image Filename',
+                'type' => 'text',
+            ],
+            [
+                'name' => 'filename_met_station',
+                'label' => 'Met Station Photo Filename',
+                'type' => 'text',
+            ],
+            [
+                'name' => 'filename_nearby_village',
+                'label' => 'Nearby Village Filename',
+                'type' => 'text',
+            ],
+            
```

Comment:
Do we definitely need these 3 images? Or would it work better as an upload_multiple field? (I don't know the answer - just asking as my assumption was an upload_multiple would work, and then the Vue component could render each one with a v-for loop. But if we definitely need these 3 image types for each station, we should keep the 3 fields and make these upload fields (check backpack docs)).